CMakeLists.txt

```diff
@@ -151,6 +151,10 @@ if(MSVC)
   set(CMAKE_MSVCIDE_RUN_PATH "\$(SolutionDir)/3rd-party/ltdl/\$(Configuration);\$(SolutionDir)/src/\$(Configuration);\$(SolutionDir)/src/mc_control/fsm/\$(Configuration)")
 endif(MSVC)
 
+if(WIN32)
+  add_definitions(/bigobj)
```

Comment:
maybe it should only be added at the target level for mc_rtc libraries?

e.g. in the `install_mc_rtc_lib` macro

```cmake
if(MSVC)
  target_compile_options(${MC_LIB} PRIVATE "/bigobj")
endif()
```

although maybe it's needed to build some of the tests as well?

Reply:
Done in 0e665f5.
Also switched the test from `if(WIN32)` to `if(MSVC)` in `install_mc_rtc_lib_common`.